lib/p5.play.js

```diff
@@ -2508,7 +2519,7 @@ function Sprite(pInst, _x, _y, _w, _h) {
           push();
           tint(this.tint);
         }
-        if(this.alpha) {
+        if(this.alpha < 1) {
```

Comment:
i don't think i understand this change -- why only call `alphaTint` if `this.alpha < 1`?

Reply:
An alpha of 1 is fully opaque, so running `alphaTint` in that case will have no effect. Since 1 is also the default value, in a project where the student isn't using alpha at all, this will keep us from running alphaTint unnecessarily.

Reply:
ahh, okay, that makes sense. are there any weird edge cases we need to double-check here? the main one i think of: what happens when a student changes alpha from < 1 to 1? is the sprite rendered as expected?

Reply:
No edge cases that I know of. I also added a check that makes sure the value of alpha is between  0 and 1, which should cover a lot of possible edge cases. For the one you mentioned, that won't be a problem since this is run as part of rendering the sprite and alphaTint applies the transparency - when the alpha is <1, the alphaTint is called and and the image is transparent. If the next time the image is rendered, alpha is 1, it doesn't call alphaTint, so no transparency is applied and it is opaque.